src/desktop.md

```diff
@@ -372,43 +414,61 @@ In addition to that executable, you need the following:
+your macOS application before distributing 
 it outside of the macOS App Store. 
 
-The first step in both of the above processes involves working with your app inside of Xcode.
-To be able to compile your app from inside of Xcode you first need to build the app for release
-using the `flutter build` command, then open the Flutter macOS Runner application.
+The first step in both of the above processes
+involves working with your app inside of Xcode.
+To be able to compile your app from inside of
+Xcode you first need to build the app for release
+using the `flutter build` command, then open the
+Flutter macOS Runner application.
 
 ```bash
 $ flutter build macos
 $ open macos/Runner.xcworkspace
 ```
 
-Once inside of Xcode, follow either Apple's [documentation on notarizing macOS Applications][],
-or [on distributing an application via the App Store][]. You should also
-read through the [macOS-specific support](#macos-specific-support) section below
-to understand how entitlements, the App Sandbox, and the Hardened Runtime
+Once inside of Xcode, follow either Apple's
+[documentation on notarizing macOS Applications][],
+or [on distributing an application through the App Store][].
+You should also read through the
+[macOS-specific support](#macos-specific-support)
+section below to understand how entitlements,
+the App Sandbox, and the Hardened Runtime
 impact your distributable application.
 
-[distribute your app via the macOS App Store]: https://developer.apple.com/macos/submit/
+[distribute it through the macOS App Store]: https://developer.apple.com/macos/submit/
 [documentation on notarizing macOS Applications]: https://developer.apple.com/documentation/xcode/notarizing_macos_software_before_distribution
-[on distributing an application via the App Store]: https://help.apple.com/xcode/mac/current/#/dev067853c94
+[on distributing an application through the App Store]: https://help.apple.com/xcode/mac/current/#/dev067853c94
 
-#### Linux
+### Linux
 
-The executable can be found in your project under `build/linux/<build mode>/bundle/`. Alongside your 
-executable binary in the `bundle` directory there are two directories:
- * `lib` contains the required `.so` library files
- * `data` contains the application's data assets, such as fonts or images
+The executable can be found in your project under
+`build/linux/<build mode>/bundle/`. Alongside your 
+executable binary in the `bundle` directory there are
+two directories:
 
-In addition to these files, your application also relies on various operating system libraries that your
-application has been compiled against. You can see the full list by running `ldd` against your application.
+ * `lib` contains the required `.so` library files
+ * `data` contains the application's data assets,
+    such as fonts or images
+
+In addition to these files, your application also
+relies on various operating system libraries that your
+application has been compiled against.
```

Comment:
s/application/it/ here since there's another 'application' earlier in the sentence.